src/api/app/models/concerns/report_bug_url.rb

```diff
@@ -0,0 +1,37 @@
+module ReportBugUrl
+  extend ActiveSupport::Concern
+
+  def report_bug_or_bugzilla_url
+    return report_bug_url if Flipper.enabled?(:foster_collaboration, User.possibly_nobody) && report_bug_url.present?
+
+    return '' if ::Configuration.bugzilla_url.blank?
```

Comment:
This should be `nil` instead I think.

Reply:
It doesn't make a difference. Applied.

Reply:
Sure it does, possible returns are not nil or a link. Instead of nil, empty string or a link. Empty string and nil behave very differently in subtle ways :)